Git/README.md

```diff
@@ -20,39 +19,55 @@ git config --global user.email "your_email@youremail.com"
 
 They will get added to your `.gitconfig` file.
 
-To push code to your GitHub repositories, we're going to use the recommended
-HTTPS method (versus SSH). To prevent `git` from asking for your username and
-password every time you push a commit you can cache your credentials by running
-the following command, as described in the
-[instructions](https://help.github.com/articles/caching-your-github-password-in-git/).
+To push code to your GitHub repositories, we will use the recommended HTTPS method. There are also instructions for using SSH. To prevent `git` from asking for your username and password every time you push a commit you can cache your credentials by running the following command, as described in the [instructions](https://help.github.com/articles/caching-your-github-password-in-git/).
 
-    git config --global credential.helper osxkeychain
+```sh
+git config --global credential.helper osxkeychain
+```
+
+## Using HTTPS for GitHub (recommended)
+
+The instructions below are referenced from [the official documentation](https://help.github.com/en/github/using-git/which-remote-url-should-i-use#cloning-with-https-urls-recommended).
+
+### Clone repositories using HTTPS
+
+After creating a new repo on GitHub, clone it using:
+
+```sh
+git clone https://github.com/<username>/<repo-name>.git
+```
+
+- if you had initialized with a README.
```

Comment:
Isn't this minus sign at the start of the line going to turn this into a list? Is that intentional?

Reply:
Yep, though now you're missing a space which really should be there. In Markdown, you can escape special characters with a "\", so how about "\- if you had initialized with a README."?

Also, you can check whether your formatting works by opening your changed code in a local editor with Markdown Preview, like Atom or VS Code.

Reply:
Fixed. Apparently you escape the dash with a backslash(\)

Reply:
Oh, LOL: I had initially included the backslash in my comment there, but I forgot that GitHub comments themselves are Markdown, so it just acted as an escape and disappears when you view my comment. :D

Reply:
E.g. this is what my comment looks like to me:

```markdown
Yep, though now you're missing a space which really should be there. In Markdown, you can escape special characters with a "\", so how about "\- if you had initialized with a README."?

Also, you can check whether your formatting works by opening your changed code in a local editor with Markdown Preview, like Atom or VS Code.
```

Reply:
The same thing happened to me! I had a backslash in the brackets there:
`Fixed. Apparently you escape the dash with a backslash(\)`

Reply:
That's why I didn't understand `with a "",` in your comment

Reply:
Now it all makes sense! Sorry about that!

Reply:
No problem at all. It was pretty funny

Reply:
What's left to do now?

Reply:
Nothing, IMHO! Just wait for the maintainers to merge it.

Reply:
Okay! Thanks for guiding me through my first PR (Well technically it isn't my first one, but it's the only one in a repo that's not dead :| )

Reply:
You're welcome! Thanks for your contribution, and congratulations in advance on getting your first PR merged!